src/components/cells/cell-editor.jsx

```diff
@@ -200,6 +208,7 @@ function mapStateToProps(state, ownProps) {
     lineNumbers: true,
     keyMap: 'sublime',
     comment: codeMirrorMode === 'javascript',
+    readOnly: getReadOnly(),
```

Comment:
any reason you define this function then call it immediately? would it work just set the value of `readOnly` directly, or am i missing something?